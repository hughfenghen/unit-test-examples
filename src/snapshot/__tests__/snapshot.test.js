
function createDefaultData() {
  return {
    a: {
      aa1: 1,
      aa2: {
        aaa1: true,
        aaa2: [],
      }
    },
    b: [{
      bb1: 'bbb',
    }]
  }
}

test('snapshot', () => {
  const data = createDefaultData()
  expect(data).toMatchSnapshot()
})
