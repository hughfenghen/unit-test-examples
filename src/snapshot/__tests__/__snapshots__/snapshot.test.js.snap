// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
{
  "a": {
    "aa1": 1,
    "aa2": {
      "aaa1": true,
      "aaa2": [],
    },
  },
  "b": [
    {
      "bb1": 1,
    },
  ],
}
`;
